stacks-signer/src/signerdb.rs

```diff
@@ -160,12 +160,20 @@ pub struct BlockInfo {
     pub state: BlockState,
     /// Consumed processing time in milliseconds to validate this block
     pub validation_time_ms: Option<u64>,
+    /// Wether the block is a tenure change block
+    pub tenure_change: bool,
```

Comment:
Do you specifically want to consider `BlockFound` tenure changes, or both `BlockFound` and `Extended` tenure changes?  The code currently does the latter.

Reply:
Blockfound and tenure extend are both contained within TenureChange payload. So it does check both. (inside TenureChangePayload contains a cause: TenureChangeCause which says whether blockfound or tenure extend)

Reply:
Yes, I'm aware :)  I was asking to confirm that this `tenure_change` flag ought to be true in the `Extended` case.